imposter.go

```diff
@@ -309,6 +311,17 @@ type Response struct {
 	Type string
 	// Value is the value of the Response; either of type HTTPResponse or TCPResponse.
 	Value interface{}
+	// Behaviors is an optional field allowing the user to define response behavior.
+	Behaviors *Behaviors
+}
+
+// Behaviors defines the possible response behaviors for a stub.
+// Currently supported values are:
+// wait (milliseconds)
```

Comment:
It might be good to leave a more detailed description of this behaviour. Consider just copying the description from the official docs:

> Adds latency to a response by waiting a specified number of milliseconds before sending the response.